HealthcareSystem/EventSourcingService/Model/StatisticEvent.cs

```diff
@@ -0,0 +1,27 @@
+﻿using EventSourcingService.Model.Enum;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EventSourcingService.Model
+{
+    public class StatisticEvent : DomainEvent
+    {
+        public int SessionId { get; }
```

Comment:
What exactly is the session id though? I don't remember anything that would match up in your session information?

Reply:
Explained in person, session id didn't exist yet and will need to be generated.